docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -12,20 +12,39 @@ include::intro.adoc[]
 
 include::features.adoc[]
 
-== Introduction to Brave
+== Introduction
+
+Spring Cloud Sleuth is a layer over https://github.com/openzipkin/brave[Brave].
 
 Brave is a distributed tracing instrumentation library. Brave typically
 intercepts production requests to gather timing data, correlate and propagate
-trace contexts. While typically trace data is sent to Zipkin server,
-third-party plugins are available to send to alternate services such as Amazon
-X-Ray.
-
-Spring Cloud Sleuth is a layer over https://github.com/openzipkin/brave[Brave].
-It configures everything you need to get started with tracing. Sleuth
+trace contexts.
+
+Trace data, also called spans, are typically reported to https://zipkin.io[Zipkin].
+Zipkin is an Open Source tracing system, which includes a UI and various
+collectors, such as HTTP and messaging.
+
+Many Open Source and commercial products accept https://zipkin.io/zipkin-api/#/default/post_spans[Zipkin format].
+Some options are documented https://zipkin.io/pages/extensions_choices.html[here],
+but many are not. If you cannot use Zipkin and your product isn't listed, clarify
+with your support representative and have them update that page. In many cases,
+products already support Zipkin format, they just don't document it.
+
+Traces connect from host to host using header propagation. The default format is
```

Comment:
I wonder if "service to service" is better than host to host as it uses the vocab of tracing and doesn't imply a service has to be a different host.